libkernelflinger/vars.c

```diff
@@ -254,32 +254,70 @@ static void set_provisioning_mode(BOOLEAN provisioning)
 	current_state = provisioning ? UNLOCKED : LOCKED;
 }
 
-enum device_state get_current_state()
+static EFI_STATUS read_efi_device_state(UINT8 *state)
```

Comment:
I would name this parameter `state_p` and the internal variable `state` instead of `stored_state`

Reply:
stored_state is got from the old code.